shiny/observatory/app.R

```diff
@@ -1,6 +1,88 @@
 library(shiny)
+library(tidyverse)
+library(readxl)
 library(r2d3)
 
+colName <- "hostname"
```

Comment:
This code should go at the top of the R file for this tab/visualisation (but keep the library imports in app.R).

Reply:
will move to the tab